fridgify/lib/service/user_service.dart

```diff
@@ -115,11 +123,14 @@ class UserService {
     throw new FailedToFetchContentException();
   }
 
+  // TODO: Refactor this. To check for an error you threw yourself is a bad
```

Comment:
@DennisRein This should be refactored. I dont think you should throw exceptions just to catch them yourself. I would call this a bad practice. Just return what is unique and what isnt unique